### ISSUE TITLE
Migrating pyside2 to pyside6 and making use release candidates 

### ISSUE THREAD

**Issue by mxochicale** [SciKit-Surgery/scikit-surgerytutorial02#33]
This issue will track the migration of pyside2 to pyside6 with the use release candidates.

**Comment by mxochicale** [comment]
Not sure why I don't see https://github.com/SciKit-Surgery/scikit-surgerytutorial02/blob/master/doc/04_Planning.rst in https://scikit-surgerytutorial02.readthedocs.io/en/latest/04_Planning.html?

**Comment by thompson318** [comment]
I don't understand. It's there for me.

**Comment by mxochicale** [comment]
Hi Steve, 

In the master/doc/04_Planning.rst from the repo we have "Also consider to read [Goals of Good Software Design](https://github.com/lorensen/VTKExamples/blob/master/src/VTKBook/02Chapter2.md#22-goals-of-good-software-design) which should be robust, understandable, extendible, modular, maintainable, and reusable." but the actual readthedocs site don't show that update. Am I missing something to update the online readthedocs? 

Thanks,
Miguel

**Comment by thompson318** [comment]
It's because the read the docs build is failing, see attached image. Can you access the following link?
https://readthedocs.org/projects/scikit-surgerytutorial02/builds/20312864/
Do you have a read the docs account? We should add you as a maintainer.
![Untitled](https://github.com/SciKit-Surgery/scikit-surgerytutorial02/assets/2535165/6e9eb589-3024-46b6-8b3b-f11006d56d5c)

**Comment by mxochicale** [comment]
Yes I can access the link and the problems seems to be with python3.7 -mvirtualenv and PySide6 versions but not sure where to update versions.

My readthedocs account is "mxochicale (Miguel Xochicale)".
Thanks, Miguel

**Comment by thompson318** [comment]
I've invited you to the maintainers list. I don't know what the problem is yet.
